### PR TITLE
[zh-cn] Resync the flow control concept page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -88,8 +85,7 @@ API 优先级与公平性（APF）特性由特性门控控制，默认情况下
 有关特性门控的一般性描述以及如何启用和禁用特性门控，
 请参见[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
 APF 的特性门控称为 `APIPriorityAndFairness`。
-此特性也与某个 {{< glossary_tooltip term_id="api-group" text="API 组" >}}
-相关：
+此特性也与某个 {{< glossary_tooltip term_id="api-group" text="API 组" >}} 相关：
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:88]:
```suggestion
此特性也与某个 {{< glossary_tooltip term_id="api-group" text="API 组" >}}相关：
```

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 2]
```diff
@@ -129,13 +125,14 @@ causing failed requests when the average load is acceptably low.
 ## 概念    {#concepts}
 
 APF 特性包含几个不同的功能。
-传入的请求通过 _FlowSchema_ 按照其属性分类，并分配优先级。
+传入的请求通过 **FlowSchema** 按照其属性分类，并分配优先级。
 每个优先级维护自定义的并发限制，加强了隔离度，这样不同优先级的请求，就不会相互饿死。
-在同一个优先级内，公平排队算法可以防止来自不同 _flow_ 的请求相互饿死。
+在同一个优先级内，公平排队算法可以防止来自不同 **流（Flow）** 的请求相互饿死。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:130]:
```suggestion
在同一个优先级内，公平排队算法可以防止来自不同**流（Flow）**的请求相互饿死。
```

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 3]
```diff
@@ -148,7 +145,7 @@ concurrent requests as its configuration allows.
 
 如果未启用 APF，API 服务器中的整体并发量将受到 `kube-apiserver` 的参数
 `--max-requests-inflight` 和 `--max-mutating-requests-inflight` 的限制。
-启用 APF 后，将对这些参数定义的并发限制进行求和，然后将总和分配到一组可配置的 _优先级_ 中。
+启用 APF 后，将对这些参数定义的并发限制进行求和，然后将总和分配到一组可配置的 **优先级** 中。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:148]:
```suggestion
启用 APF 后，将对这些参数定义的并发限制进行求和，然后将总和分配到一组可配置的**优先级**中。
```

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 4]
```diff
@@ -185,8 +182,8 @@ many instances should authenticate with distinct usernames
 （尤其是在一个较为常见的场景中，一个有故障的客户端会疯狂地向 kube-apiserver 发送请求，
 理想情况下，这个有故障的客户端不应对其他客户端产生太大的影响）。
 公平排队算法在处理具有相同优先级的请求时，实现了上述场景。
-每个请求都被分配到某个 _流_ 中，该 _流_ 由对应的 FlowSchema 的名字加上一个
-_流区分项（Flow Distinguisher）_ 来标识。
+每个请求都被分配到某个 **流（Flow）** 中，该 **流** 由对应的 FlowSchema 的名字加上一个
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:185]: Not sure should we have a space around **bold** or not.

**Review by Sea-n — COMMENTED** [review]
Looks good to me.
Some non-blocking nits.

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md (hunk 4 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/cluster-administration/flow-control.md:185]: I'm not sure about it either. Putting extra spaces around is the safest way IMO, given that we have heard defects on this before.

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md (hunk 2 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/cluster-administration/flow-control.md]: I'm gonna delete these spaces and see how the output is rendered.

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md (hunk 4 above)
Comment (Sea-n) [reply content/zh-cn/docs/concepts/cluster-administration/flow-control.md:185]: For sentence like `中文**(半形括號)**` will cause not-formatting visual bug.
But I'm sure in `中文**粗體**無空格` syntax it's fine.

Anyways, it's not bad to add a space around.

**Comment by tengqm** [comment]
Looks like the spaces are strictly needed.
Screen Shot 2022-06-27 at 3 19 58 PM

**Pull request merged**